packages/devtools_app/lib/src/shared/console/eval/eval_service.dart

```diff
@@ -60,6 +60,19 @@ class EvalService extends DisposableController with AutoDisposeControllerMixin {
     return await _service.getObject(ref, objRef.id!);
   }
 
+  Future<Response> evalInRunningApp(
```

Comment:
Can you add some dart doc to this describing that this evaluation occurs in the isolate's root library?